pennylane/pauli/pauli_arithmetic.py

```diff
@@ -120,6 +120,11 @@ def __copy__(self):
         """Copy the PauliWord instance."""
         return PauliWord(dict(self.items()))
 
+    def __deepcopy__(self, memo):
+        res = self.__copy__()
+        memo[id(self)] = res
```

Comment:
What was forcing this?

Reply:
Since `_pauli_rep` is an attribute, it also gets copied when copying the Operator class. There were certain tests failing because instances of deep copied ops would call the deep copy method of the PauliWord or PauliSentence which was not defined.

Reply:
I believe using a `_pauli_rep` property would fix this.